CHANGES.rst

```diff
@@ -5,6 +5,11 @@ Changes for Crate Admin Interface
 Unreleased
 ==========
 
+Breaking Changes
```

Comment:
I think this could go as fix within `1.15`, given that `1.15.0` already broke compatibility and is for 4.0+

Reply:
ok, that would be then version 1.15.1 ?